src/main/scala/com/github/funprog/funbot/DiceBot.scala

```diff
@@ -3,29 +3,63 @@ package com.github.funprog.funbot
 import scala.util.Random
 
 class DiceBot(val random: Random) {
```

Comment:
Do we have any reason get this random object from callee? This question goes to @jwChung